hazelcast/src/main/java/com/hazelcast/internal/probing/ProbeRegistryImpl.java

```diff
@@ -0,0 +1,681 @@
+
+        // render cycle state
+        private ProbeRenderer renderer;
+        private ProbeLevel level;
+
+        ProbingCycleImpl(Collection<ProbeSourceEntry> sources) {
+            this.sources = sources;
+        }
+
+        @Override
+        public void renderAt(ProbeLevel level, ProbeRenderer renderer) {
+            this.level = level;
+            this.renderer = renderer;
+            openContext(); // reset
+            for (ProbeSourceEntry entry : sources) {
+                if (isProbed(entry.updateLevel)) {
+                    entry.updateIfNeeded();
+                }
+                try {
+                    openContext(); // just to be sure it is done even if ommitted in the source
+                    entry.source.probeIn(this);
+                } catch (Exception e) {
+                    LOGGER.warning("Exception while probing source "
+                            + entry.source.getClass().getSimpleName(), e);
+                }
+            }
+        }
+
+        @Override
+        public void probe(final ProbeLevel level, Object instance, final String... methods) {
+            if (instance == null || !isProbed(level)) {
+                return;
+            }
+            PROBE_METADATA.computeIfAbsent(instance.getClass(), new Function<Class<?>, ProbeAnnotatedType>() {
+
+                @Override
+                public ProbeAnnotatedType apply(Class<?> type) {
+                    return new ProbeAnnotatedType(type, level, methods);
+                }
+            }).probeIn(this, this.level, null, instance);
+        }
+
+        @Override
+        public void probe(Object instance) {
+            probe(null, instance);
+        }
+
+        @Override
+        public void probe(CharSequence prefix, Object instance) {
+            if (instance == null) {
+                return;
+            }
+            PROBE_METADATA.computeIfAbsent(instance.getClass(), 
+                    new Function<Class<?>, ProbeAnnotatedType>() {
+
+                @Override
+                public ProbeAnnotatedType apply(Class<?> type) {
+                    //TODO use EMPTY constant or null for types that do not have probes at all
+                    return new ProbeAnnotatedType(type);
+                }
+            }).probeIn(this, level, prefix, instance);
+        }
+
+        @Override
+        public void probe(Object[] instances) {
+            if (instances != null) {
+                for (int i = 0; i < instances.length; i++) {
+                    probe(instances[i]); 
+                }
+            }
+        }
+
+        @Override
+        public void probe(CharSequence name, long value) {
+            probe(ProbeLevel.MANDATORY, name, value);
+        }
+
+        @Override
+        public void probe(CharSequence name, double value) {
+            probe(ProbeLevel.MANDATORY, name, value);
+        }
+
+        @Override
+        public void probe(CharSequence name, boolean value) {
```

Comment:
Probes are created by default under info; so it seems logical that probe would imply info level as well.

Reply:
Change default to `INFO` to be consistent.